assets/package.json

```diff
@@ -93,6 +93,7 @@
     "luxon": "1.21.3",
     "matchmedia-polyfill": "0.3.2",
     "normalizr": "3.4.1",
+    "raf-throttle": "2.0.3",
```

Comment:
Added the throttling to prevent unnecessary rerenderings

Reply:
Guess I need to look up throttling, are you trying to speed things up?

Reply:
I'm trying to prevent the javascript from trying to rerender except when it is time for the browser to do so. It should make the browser less busy and maybe speed up rendering of the page. You might want to look at window.requestAnimationFrame as well.